### PR TITLE
Merge Wazuh Dashboard v3 

### PR DESCRIPTION
**Pull request by c-bordon**
|Related issue|
|---|
|https://github.com/wazuh/wazuh-packages/issues/1258|

## Description

- Wazuh plugin download path updated
- Changed error message from "Opensearch Dashboards server is not ready yet" to "Wazuh Dashboard server is not ready yet"
- A validation is added so that, in case of an upgrade, the Keystore is not replaced

## Logs example

```
{"type":"response","@timestamp":"2022-02-15T12:24:30Z","tags":[],"pid":18232,"method":"get","statusCode":200,"req":{"url":"/ui/wazuh_wazuh_bg.svg","method":"get","headers":{"host":"192.168.56.252","connection":"keep-alive","user-agent":"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.87 Safari/537.36","accept":"image/avif,image/webp,image/apng,image/svg+xml,image/*,*/*;q=0.8","sec-gpc":"1","sec-fetch-site":"same-origin","sec-fetch-mode":"no-cors","sec-fetch-dest":"image","referer":"https://192.168.56.252/ui/legacy_light_theme.css","accept-encoding":"gzip, deflate, br","accept-language":"en-US,en;q=0.9"},"remoteAddress":"192.168.56.1","userAgent":"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.87 Safari/537.36","referer":"https://192.168.56.252/ui/legacy_light_theme.css"},"res":{"statusCode":200,"responseTime":5,"contentLength":9},"message":"GET /ui/wazuh_wazuh_bg.svg 200 5ms - 9.0B"}
{"type":"log","@timestamp":"2022-02-15T12:37:24Z","tags":["info","plugins-system"],"pid":18232,"message":"Stopping all plugins."}
{"type":"log","@timestamp":"2022-02-15T12:37:57Z","tags":["info","plugins-service"],"pid":18398,"message":"Plugin \"visTypeXy\" is disabled."}
{"type":"log","@timestamp":"2022-02-15T12:37:57Z","tags":["info","plugins-system"],"pid":18398,"message":"Setting up [45] plugins: [alertingDashboards,usageCollection,opensearchDashboardsUsageCollection,opensearchDashboardsLegacy,mapsLegacy,share,opensearchUiShared,embeddable,legacyExport,expressions,data,home,console,apmOss,management,indexPatternManagement,advancedSettings,savedObjects,securityDashboards,indexManagementDashboards,anomalyDetectionDashboards,dashboard,notebooksDashboards,visualizations,visTypeTable,visTypeVega,visTypeTimeline,timeline,visTypeMarkdown,tileMap,regionMap,inputControlVis,visualize,ganttChartDashboards,queryWorkbenchDashboards,traceAnalyticsDashboards,reportsDashboards,charts,visTypeVislib,visTypeTimeseries,visTypeTagcloud,visTypeMetric,discover,savedObjectsManagement,bfetch]"}
{"type":"log","@timestamp":"2022-02-15T12:37:57Z","tags":["info","savedobjects-service"],"pid":18398,"message":"Waiting until all OpenSearch nodes are compatible with OpenSearch Dashboards before starting saved objects migrations..."}
{"type":"log","@timestamp":"2022-02-15T12:37:57Z","tags":["info","savedobjects-service"],"pid":18398,"message":"Starting saved objects migrations"}
{"type":"log","@timestamp":"2022-02-15T12:37:57Z","tags":["info","plugins-system"],"pid":18398,"message":"Starting [45] plugins: [alertingDashboards,usageCollection,opensearchDashboardsUsageCollection,opensearchDashboardsLegacy,mapsLegacy,share,opensearchUiShared,embeddable,legacyExport,expressions,data,home,console,apmOss,management,indexPatternManagement,advancedSettings,savedObjects,securityDashboards,indexManagementDashboards,anomalyDetectionDashboards,dashboard,notebooksDashboards,visualizations,visTypeTable,visTypeVega,visTypeTimeline,timeline,visTypeMarkdown,tileMap,regionMap,inputControlVis,visualize,ganttChartDashboards,queryWorkbenchDashboards,traceAnalyticsDashboards,reportsDashboards,charts,visTypeVislib,visTypeTimeseries,visTypeTagcloud,visTypeMetric,discover,savedObjectsManagement,bfetch]"}
{"type":"log","@timestamp":"2022-02-15T12:37:57Z","tags":["listening","info"],"pid":18398,"message":"Server running at https://0.0.0.0:443"}
{"type":"log","@timestamp":"2022-02-15T12:37:58Z","tags":["info","http","server","OpenSearchDashboards"],"pid":18398,"message":"http server running at https://0.0.0.0:443"}


```

## Tests

- Build the package in any supported platform
  - [x] Linux
  - [ ] Windows
  - [ ] macOS
  - [ ] Solaris
  - [ ] AIX
  - [ ] HP-UX
- [x] Package installation
- [x] Package upgrade
- [ ] Package downgrade
- [x] Package remove
- [x] Package install/remove/install
- [ ] Change added to CHANGELOG.md

- Tests for Linux RPM
  - [x] Build the package for x86_64
  - [ ] Build the package for i386
  - [ ] Build the package for armhf
  - [ ] Build the package for aarch64
  - [ ] `%files` section is correctly updated if necessary
- Tests for Linux deb
  - [x] Build the package for x86_64
  - [ ] Build the package for i386
  - [ ] Build the package for armhf
  - [ ] Build the package for aarch64
  - [ ] Package install/remove/install
  - [ ] Package install/purge/install
  - [ ] Check file permissions after installing the package
- Tests for macOS
  - [ ] Test the package from macOS Sierra to Mojave
- Tests for Solaris
  - [ ] Test the package on Solaris 10
  - [ ] Test the package on Solaris 11
  - [ ] Check file permissions on Solaris 11 template
- Tests for IBM AIX
  - [ ] `%files` section is correctly updated if necessary
  - [ ] Check the changes from IBM AIX 5 to 7

### CODE REVIEW

File: stack/dashboard/deb/debian/rules [hunk 1]
```diff
@@ -114,7 +115,7 @@ override_dh_install:
 	chown $(USER):$(GROUP) $(TARGET_DIR)/etc/systemd/system/$(NAME)
 	chown $(USER):$(GROUP) $(TARGET_DIR)/etc/default/$(NAME)
 
-	runuser $(USER) --shell="/bin/bash" --command="$(TARGET_DIR)$(INSTALLATION_DIR)/bin/opensearch-dashboards-plugin install https://s3.amazonaws.com/warehouse.wazuh.com/stack/dashboard/wazuh-1.2.0.zip" 
+	runuser $(USER) --shell="/bin/bash" --command="$(TARGET_DIR)$(INSTALLATION_DIR)/bin/opensearch-dashboards-plugin install https://s3.us-west-1.amazonaws.com/packages-dev.wazuh.com/pre-release/ui/dashboard/wazuh-$(BASE_VERSION).zip"
```
Comment (okynos) [inline stack/dashboard/deb/debian/rules:118]: Remove `s3.us-west-1.amazonaws.com/` from URL, not needed

**Review by okynos — CHANGES_REQUESTED** [review]
Little change, gj.

**Review by alberpilot — APPROVED** [review]
With @okynos suggestions, lgtm

File: stack/dashboard/deb/debian/rules (hunk 1 above)
Comment (c-bordon) [reply stack/dashboard/deb/debian/rules]: Done: https://github.com/wazuh/wazuh-packages/commit/91a20ec9483d27a285693371697c2f66a3451154

**Review by okynos — APPROVED** [review]
LGTM

**Pull request merged**